subprojects/test-runner/device-provider/impl/src/main/kotlin/com/avito/android/runner/devices/internal/kubernetes/RemoteDeviceProvider.kt

```diff
@@ -65,15 +65,19 @@ internal class RemoteDeviceProviderImpl(
             val podIp = ip
             requireNotNull(podIp) { "Pod: $name must have an IP" }
 
-            val serial = emulatorSerialName(name = podIp)
+            val name = portForward?.let { portForward ->
```

Comment:
I'm defenetly think that If this change would be done it should be done with new layer of absctraction. For example `DevicePortProvider`